public/index.html

```diff
@@ -218,8 +218,8 @@ <h1>Order Summary</h1>
         <p class="label">Shipping</p>
         <p class="price">Free</p>
       </div>
-      <div class="line-item demo">
-        <div id="demo">
+      <div id="testmode-info" class="line-item">
```

Comment:
Move the demo class down one -- Since we don't use the `demo` as an ID anywhere besides the CSS. This makes it easy to reuse the style for for a dialog that shows the integration live trace link